ownCloud/Viewer/ImageDisplayViewController.swift

```diff
@@ -38,12 +54,59 @@ class ImageDisplayViewController : DisplayViewController {
 		}
 	}
 
+	// MARK: - Frame changes
 	override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
 		super.viewWillTransition(to: size, with: coordinator)
 		self.scrollView?.updateScaleForRotation(size: size)
 	}
+
+	// MARK: - Gesture recognizers actions.
+	@objc func tapToZoom() {
+		guard let scrollView = scrollView else {
+			return
+		}
+
+		if scrollView.zoomScale != scrollView.minimumZoomScale {
+			scrollView.setZoomScale(scrollView.minimumZoomScale, animated: true)
+		} else {
+			scrollView.setZoomScale(scrollView.maximumZoomScale / 3, animated: true)
+		}
+
+		setNeedsUpdateOfHomeIndicatorAutoHidden()
```

Comment:
@pablocarmu method name is not self-explanatory for me

Reply:
This is from UIKit, it asks for the value of the `prefersHomeIndicatorAutoHidden` and update the home indicator preferences.

[setNeedsUpdateOfHomeIndicatorAutoHidden](https://developer.apple.com/documentation/uikit/uiviewcontroller/2887509-setneedsupdateofhomeindicatoraut#)

Reply:
@pablocarmu Ok, I was not aware that it is Apple's API. Sorry:) My bad.